### PR TITLE
fix eslint errors in SelectableCard, Slider and ToastProvider

### PR DESCRIPTION
**Pull request by thayannevls**
#### What is the purpose of this pull request?
#trivial fix Eslint errors #1082 

#### How should this be manually tested?
`yarn & yarn start`
`yarn lint`

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by emersonlaurentino** [comment]
@thayannevls you are doing a great job, thanks for the contribution. But the next time, tell us in #1082 which component you are fixing so that we have greater control and someone does not fix the same component.

**Comment by thayannevls** [comment]
Sorry @emersonlaurentino ! Next time I'll do it :)

File: react/components/ToastProvider/ToastManager.js [hunk 1]
```diff
@@ -100,15 +100,15 @@ export default class ToastManager extends Component {
       windowBounds
 
     if (this.container.current) {
```
Comment (emersonlaurentino) [inline react/components/ToastProvider/ToastManager.js]: same
```suggestion
    if (!this.container.current) {
```

File: react/components/Slider/index.js [hunk 2]
```diff
@@ -93,9 +93,10 @@ export default class Slider extends Component {
     window.removeEventListener('resize', this.updateLayout)
 
     if (this.cancelDragEvent_) {
```
Comment (emersonlaurentino) [inline react/components/Slider/index.js]: the condition says that if `this.cancelDragEvent_` exists then run `this.cancelDragEvent _ ()` and `this.cancelDragEvent_ = undefined` and then return.

the way you did it, it's the other way around!

The correct should be:
```suggestion
    if (!this.cancelDragEvent_) {
```

Comment (thayannevls) [reply react/components/Slider/index.js]: My bad didn't notice it :) fixed it, thanks!

**Pull request merged**